wporg-gp-translation-events.php

```diff
@@ -96,8 +99,59 @@ public function activate() {
 		PRIMARY KEY (`translate_event_actions_id`),
 		UNIQUE KEY `event_per_translated_original_per_user` (`event_id`,`locale`,`original_id`,`user_id`)
 		) COMMENT='Tracks translation actions that happened during a translation event'";
-		require_once ABSPATH . 'wp-admin/includes/upgrade.php';
-		dbDelta( $create_table );
+		dbDelta( $create_actions_table );
+
+		// Create attendees table.
+		$create_attendees_table = "
+		CREATE TABLE `{$gp_table_prefix}event_attendees` (
+			`translate_event_attendees_id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+			`event_id` int(10) NOT NULL COMMENT 'Post_ID of the translation_event post in the wp_posts table',
+			`user_id` int(10) NOT NULL COMMENT 'ID of the user who is attending the event',
+		PRIMARY KEY (`translate_event_attendees_id`),
+		UNIQUE KEY `event_per_user` (`event_id`,`user_id`),
+		INDEX `user` (`user_id`)
+		) COMMENT='Attendees of events'";
+		dbDelta( $create_attendees_table );
+
+		// TODO: Remove this once it has been run in production.
+		try {
+			// Don't run it during tests.
+			if ( 'true' === getenv( 'TRANSLATION_EVENTS_NO_IMPORT_LEGACY_ATTENDEES' ) ) {
```

Comment:
If you want to run this once in production and to avoid running on the tests, I think you need to invert the check:

```suggestion
			if ( 'true' !== getenv( 'TRANSLATION_EVENTS_NO_IMPORT_LEGACY_ATTENDEES' ) ) {
```

Reply:
Nice catch, fixed